batch/batch/front_end/front_end.py

```diff
@@ -1417,13 +1417,14 @@ async def on_startup(app):
 
     row = await db.select_and_fetchone(
         '''
-SELECT worker_type, worker_cores, worker_disk_size_gb,
+SELECT worker_type, worker_cores, worker_disk_size_gb, standing_worker_cores,
   instance_id, internal_token, n_tokens FROM globals;
 ''')
 
     app['worker_type'] = row['worker_type']
     app['worker_cores'] = row['worker_cores']
     app['worker_disk_size_gb'] = row['worker_disk_size_gb']
+    app['standing_worker_cores'] = row['standing_worker_cores']
```

Comment:
This will become out-of-date if someone edits this on the driver. It also appears unused in the front-end?